public/css/style.css

```diff
@@ -28,6 +28,13 @@
     height: 150px;
 }
 
+.selected-image-display {
+    margin: 1px 1px -3px 1px;
+    padding: 0 0 0 0;
+    width: 300px;
+    height: 300px;
+}
+
```

Comment:
Just a comment for my own enlightenment: is this a matter of trial and error? Or is there a method to the madness? e.g. a commonly-cited UI design manual?